types/tlf-log/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "export-just-namespace": false
```

Comment:
Could you please enable this rule? Unless you have a specific need for it.

Reply:
Whoops! Forgot to remove that before pushing. Fixed in 6a192a396cae10eeef084fd4d6e6d9c2dee86433.